### ISSUE TITLE
Direct message window does not render in high contrast

### ISSUE THREAD

**Issue by rkingett** [manuelcortez/TWBlue#75]
The send message dialogue box just shows up as a black square in high contrast mode and I MUST use a screen reader to navigate around the box
